submissions/Lu-sure/Popup/index.html

```diff
@@ -0,0 +1,168 @@
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/mail.png" alt="Mail icon" class="popup-item-img">
+                      Mail
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/news.png" alt="News icon" class="popup-item-img">
+                      News
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/maps.png" alt="Map icon" class="popup-item-img">
+                      Map
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/files.png" alt="Files icon" class="popup-item-img">
+                      Files
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/contacts.png" alt="Contacts icon" class="popup-item-img">
+                      Contacts
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/tanslation.png" alt="Translater icon" class="popup-item-img">
+                      Translate
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/videos.png" alt="Videos icon" class="popup-item-img">
+                      Videos
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/search.png" alt="Search icon" class="popup-item-img">
+                      Search
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/photos.png" alt="Photo icon" class="popup-item-img">
+                      Photo
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/apps.png" alt="Apps icon" class="popup-item-img">
+                      Apps
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/documents.png" alt="Documents icon" class="popup-item-img">
+                      Documents
+                    </a>
+                  </li>
+                </ul>
+                
+                <input type="checkbox" id="more-button" class="hide-checkbox more" tabindex="1">
```

Comment:
All interactive elements have a focus by default, use `tabindex` attribute for adding a noninteractive element to the focusable elements or for changing focus order. [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex).